editor/animation_track_editor.cpp

```diff
@@ -2698,7 +2698,11 @@ void AnimationTrackEdit::gui_input(const Ref<InputEvent> &p_event) {
 						Node *nd = ap->get_node(ap->get_root())->get_node(NodePath(npath.get_concatenated_names()));
 						StringName prop = npath.get_concatenated_subnames();
 						PropertyInfo prop_info;
-						ClassDB::get_property_info(nd->get_class(), prop, &prop_info);
+
```

Comment:
This should be inside an if statement instead, like so:
```cpp
						if (nd) {
							StringName prop = npath.get_concatenated_subnames();
							PropertyInfo prop_info;
							ClassDB::get_property_info(nd->get_class(), prop, &prop_info);

							bool is_angle = prop_info.type == Variant::FLOAT && prop_info.hint_string.find("radians") != -1;
							if (is_angle) {
								menu->add_icon_item(get_editor_theme_icon(SNAME("InterpLinearAngle")), TTR("Linear Angle"), MENU_INTERPOLATION_LINEAR_ANGLE);
								menu->add_icon_item(get_editor_theme_icon(SNAME("InterpCubicAngle")), TTR("Cubic Angle"), MENU_INTERPOLATION_CUBIC_ANGLE);
							}
						}
```
Starting from the line below `Node *nd = ap->get_node(ap->get_root())->get_node(NodePath(npath.get_concatenated_names()));`

Reply:
I have ammended my commit with the recommendations you provided

Reply:
Looks good! But remove this line

Reply:
Done